config/config.go

```diff
@@ -535,6 +535,9 @@ type TracingConfig struct {
 	SamplingFraction float64           `yaml:"sampling_fraction,omitempty"`
 	Insecure         bool              `yaml:"insecure,omitempty"`
 	TLSConfig        config.TLSConfig  `yaml:"tls_config,omitempty"`
+	Headers          map[string]string `yaml:"headers,omitempty"`
```

Comment:
Should we have this as 

```
Headers struct {
Authorization config.Secret `yaml:"authorization,omitempty"`
map[string]string
}
```
To avoid leaking that specific header in the UI?